tutorials/R/download-explore/NEON-download-explore.Rmd

```diff
@@ -0,0 +1,379 @@
+---
+syncID: 5f9c4048a27749c19ee8ecfc78806363
+title: "Download and Explore NEON Data"
+description: "Tutorial for downloading data from the Data Portal and the neonUtilities package, then exploring and understanding the downloaded data"
+dateCreated:  2018-11-07
+authors: [Claire K. Lunch]
+contributors: [Christine Laney, Megan A. Jones]
+estimatedTime: 1 - 2 hours
+packagesLibraries: [devtools, geoNEON, neonUtilities, rhdf5, raster]
+topics: data-management, rep-sci
+languagesTool: R, API
+dataProduct:
+code1: R/download-explore/NEON-download-explore
+tutorialSeries: 
+urlTitle: neon-explore
```

Comment:
@cklunch Hi. Before this goes live, could you change the urlTitle to something a bit more descriptive like "download-explore-neon-data"?  I like to the have URLs match the titles as much as possible as it makes it much easier to find the correct tutorial when a link is broken.